test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -62,6 +62,8 @@ var _ = SIGDescribe("CustomResourcePublishOpenAPI [Privileged:ClusterAdmin]", fu
 		properties; client-side validation MUST reject the operations. Attempt kubectl explain; the output MUST
 		explain the custom resource properties. Attempt kubectl explain on custom resource properties; the output MUST
 		explain the nested custom resource properties.
+		Note: Release v1.24 substitutes client-side validation with server-side validation by default.
+		All validation should be the same.
```

Comment:
drop these notes and drop "client-side" from the conformance description

Reply:
/hold for updating the conformance description

Reply:
done